### DISCUSSION TITLE
Restore deleted github actions workflow run

### DISCUSSION THREAD

**Discussion by kvk5592007** [community/community#131565] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

How to restore the deleted github actions workflow runs?. Scenario is if some developer run the github action workflow to deploy the code to some environment and deleted that run to hide the details. It is going to be a high risk if not executable code is deployed to critical environment. For audit purposes all the github action workflow run details are required for tracking purposes.

**Comment by Trieusivien** [comment]
> ### Select Topic Area
> Question
> 
> ### Body
> How to restore the deleted github actions workflow runs?. Scenario is if some developer run the github action workflow to deploy the code to some environment and deleted that run to hide the details. It is going to be a high risk if not executable code is deployed to critical environment. For audit purposes all the github action workflow run details are required for tracking purposes.
